### DISCUSSION TITLE
Click on graph and get the data closest to it

### DISCUSSION THREAD

**Discussion by hbina** [pyqtgraph/pyqtgraph#2697] [Q&A]
Is something like this possible? I tried working with the x and y axes but since you can move around and zoom freely within the scene i am not sure how i can calculate it

**Comment by ntjess** [comment]
See [GraphicsScene](https://pyqtgraph.readthedocs.io/en/latest/api_reference/graphicsscene/graphicsscene.html) documentation, you can listen for `sigMouseClicked`, get the `pos()`, and calculate which data point has the min distance to this pos. Does this sound like what you're after?
